src/vmm/tests/integration_tests.rs

```diff
@@ -22,12 +22,11 @@ fn default_memory_config() -> MemoryConfig {
     MemoryConfig { mem_size_mib: 1024 }
 }
 
-fn default_kernel_config() -> KernelConfig {
-    let kernel_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
-        .join("../../resources/kernel/vmlinux-hello-busybox-halt");
-    // Sanity check. Because the VMM runs in a separate process, if the file doesn't exist,
-    // all we see is a different exit code than 0.
-    assert!(kernel_path.as_path().exists());
+fn default_kernel_config(path: Option<PathBuf>) -> KernelConfig {
```

Comment:
How about instead of having `path` as an option, we pass a parameter something like image_type for which we can add a define:
```rust
enum ImageType {
    BzImage,
    Elf,
}
```

Then, we could have something like:
```rust
let kernel_config = default_kernel_config(ImageType::Elf);
// or
let kernel_config = default_kernel_config(ImageType::BzImage);
```

Reply:
I prefer having an optional `Path[Buf]` because in the future we'll likely want to run multiple kernels of the same format, so having a default one for each type of format would mean wed'd *still* have to overwrite the path for a non-default case.

Reply:
If we do have that in the future, I guess we can then have the `path` parameter as well. Right now it feels like an early optimization that is causing the path parameter to select the image type instead of only selecting the path of the binary. Alternatively we can just define `DEFAULT_ELF` and `DEFAULT_BZIMAGE` and always pass the path so that they're uniformly used.